### ISSUE TITLE
Make sure all of the data for the national calculation is of the same year

### ISSUE THREAD

**Issue by jwestw** [ONSdigital/SDG_11.2.1#128]
It's probable that the current calculation is using data from different years - e.g. 2011 (census data for disability data) and 2019 (for population estimate data). We need all the data for the calculation to be of the same year; it doesn't matter which. 

Let's start with 2019 and we can extend to other years later.

**Comment by james-westwood** [comment]
@james-westwood note to self: There's a problem in the way the `served_proportions_disagg` is calculating the served and unserved populations. This is happening in the new way that I have created the population columns by sex. The sum of the female or male population is bigger than the total population.

**Comment by james-westwood** [comment]
Checked the inputs to `served_proportions_disagg` which are the `pop_df` and the `pop_in_poly_df` and found the number of records in the  `pop_in_poly_df` is too high. It should be =<3223 records (which is the case for the pop_df) as this is the total number of OAs in Birmingham.

**Comment by james-westwood** [comment]
I have made quite a lot of progress on this ticket today.

I needed a year column, so I realised that it was probably best that I first pulled in all population data for that year. I did so for 2019.

The pop data came in xls format which took an age to read. I have created a whole_nation_pop_df which captures all of the national population figures for that year. I created a pop_year column populated with int(2019) (or whatever the year variable is). The data is stored under a folder named as that year.

Bringing in the whole nation pop data helps to solve #129
